test/utils.ts

```diff
@@ -258,3 +263,75 @@ export function getLocalEip712Signature(
     // @ts-ignore
     return sigUtil.signTypedData_v4(privateKey, { data: typedRequestData });
 }
+
+export async function deployHub(
+    penalizer: string = constants.ZERO_ADDRESS,
+    configOverride: Partial<RelayHubConfiguration> = {}
+): Promise<RelayHubInstance> {
+    const relayHubConfiguration: RelayHubConfiguration = {
+        ...defaultEnvironment.relayHubConfiguration,
+        ...configOverride
+    };
+    return await RelayHub.new(
+        penalizer,
+        relayHubConfiguration.maxWorkerCount,
+        relayHubConfiguration.minimumEntryDepositValue,
+        relayHubConfiguration.minimumUnstakeDelay,
+        relayHubConfiguration.minimumStake
+    );
+}
+
+export async function getGaslessAccount(): Promise<AccountKeypair> {
+    const a = ethWallet.generate();
+    const gaslessAccount = {
+        privateKey: a.getPrivateKey(),
+        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+        address: bufferToHex(privateToAddress(a.getPrivateKey())).toLowerCase()
+    };
+
+    return gaslessAccount;
+}
+
+export async function evmMineMany(count: number): Promise<void> {
+    for (let i = 0; i < count; i++) {
+        await evmMine();
+    }
+}
+
+export async function evmMine(): Promise<any> {
+    return await new Promise((resolve, reject) => {
+        // @ts-ignore
+        web3.currentProvider.send(
+            {
```

Comment:
why do we need to wrap the promise in another one? There seems to be no added functionality

Reply:
the `send` function is callback based function, so it doesn't return a promise unfortunately 😓

Reply:
aaah... what was I thinking here... that's clear from the impl. also I remember having this sort of wrapper in legacy marketplace

Reply:
Probably we'll solve this once we update the libraries